core/templates/components/forms/custom-forms-directives/thumbnail-uploader.component.html

```diff
@@ -24,26 +24,26 @@
 </div>
 
 <style>
-  thumbnail-uploader .thumbnail-editor {
-    height: 100px;
+  .thumbnail-editor {
+    height: 100px !important;
   }
 
-  thumbnail-uploader .oppia-editable-section {
+  .oppia-editable-section {
     border-radius: 0.25rem;
     display: table-cell;
-    height: 100px;
+    height: 120px;
     text-align: center;
     vertical-align: middle;
-    width: 180px;
+    width: 120px;
   }
 
-  thumbnail-uploader .img-thumbnail {
+  .img-thumbnail {
     border: none;
-    max-height: 100%;
-    max-width: 100%;
+    height: 120px !important;
+    width: 120px;
```

Comment:
Why did we change from 100% to 120px (Note percentage changes according to screen size, `px` doesn't)

Reply:
PTAL at the new changes